core/string/ustring.cpp

```diff
@@ -1822,7 +1822,7 @@ Error String::parse_utf8(const char *p_utf8, int p_len, bool p_skip_cr) {
 	bool decode_failed = false;
 	{
 		const char *ptrtmp = p_utf8;
-		const char *ptrtmp_limit = &p_utf8[p_len];
+		const char *ptrtmp_limit = p_len >= 0 ? &p_utf8[p_len] : NULL;
```

Comment:
```suggestion
		const char *ptrtmp_limit = p_len >= 0 ? &p_utf8[p_len] : nullptr;
```

Reply:
Thanks. Updated to include that one as well.